com.github.hneemann.digital/com.github.hneemann.digital.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "com.github.hneemann.digital",
+  "branch": "0.31",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+      "org.freedesktop.Sdk.Extension.openjdk21"
+  ],
+  "command": "Digital.sh",
+  "finish-args": [
+      "--persist=.java",
+      "--share=ipc",
+      "--share=network",
+      "--socket=x11",
+      "--socket=wayland",
+      "--filesystem=home"
+  ],
+  "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/share/aclocal",
+      "/share/man",
+      "*.la", "*.a"
+  ],
+  "build-options": {
+      "no-debuginfo": true
+  },
+  "modules": [
+        {
+            "name" : "openjdk",
+            "buildsystem" : "simple",
+            "build-commands" : [ "/usr/lib/sdk/openjdk21/install.sh" ]
+        },      
+        {
+            "name": "digital",
```

Comment:
since this is the main it should be the last in the list. dependencies are built in order.